core/src/test/java/dev/streamx/cli/ingestion/IngestionArgumentsValidationTest.java

```diff
@@ -25,7 +25,9 @@ public void shouldRejectIllegalIngestionUrl(QuarkusMainLauncher launcher) {
 
     // then
     assertThat(result.exitCode()).isNotZero();
-    assertThat(result.getErrorOutput()).contains("Illegal character in path at index 15");
+    assertThat(result.getErrorOutput()).isEqualTo(
+        "Publication endpoint URI: hattetepe:///in valid/publications/v1 is malformed. "
+            + "Illegal character in path");
```

Comment:
The change in error message is caused by other libraries being used underneath.
 - old error message came from streamx-cli's implementation of validate schema
 - new messages com from streamx's internal schema validation code